src/EnergyPlus/CurveManager.cc

```diff
@@ -1908,7 +1910,7 @@ namespace CurveManager {
 			if ( NumAlphas >= 5 ) {
 				if ( ! IsCurveOutputTypeValid( Alphas( 5 ) ) ) {
 					ShowSevereError( "GetCurveInput: For " + CurrentModuleObject + ": " + Alphas( 1 ) );
-					ShowContinueError( cAlphaFieldNames( 5 ) + " [" + Alphas( 5 ) + "] is invlaid" );
+					ShowContinueError( cAlphaFieldNames( 5 ) + " [" + Alphas( 5 ) + "] is invalid" );
```

Comment:
Pronouncing `invlaid` makes me think of the word "to blave"

![image](https://cloud.githubusercontent.com/assets/849824/24809051/8653ce64-1b7b-11e7-82f4-d653f6709078.png)